app/screens/receive-bitcoin-screen/use-receive-bitcoin.ts

```diff
@@ -369,6 +369,37 @@ export const useReceiveBitcoin = () => {
     }
   }, [pr, LL])
 
+  const receiveViaNFC = useCallback(
+    async (destination: ReceiveDestination) => {
+      if (pr?.info?.data?.invoiceType !== "Lightning" || !pr.info.data.paymentRequest) {
+        Alert.alert(LL.RedeemBitcoinScreen.error())
+        return
+      }
+
+      const { callback, k1 } = destination.validDestination
+
+      const url = `${callback}${callback.includes("?") ? "&" : "?"}k1=${k1}&pr=${
+        pr.info.data.paymentRequest
+      }`
+
+      const result = await fetch(url)
+      if (result.ok) {
+        const lnurlResponse = await result.json()
+        if (lnurlResponse?.status?.toLowerCase() !== "ok") {
+          console.error(lnurlResponse, "error with redeeming")
+          Alert.alert(LL.RedeemBitcoinScreen.redeemingError())
+          if (lnurlResponse?.reason) {
+            Alert.alert(lnurlResponse.reason)
+          }
```

Comment:
if `lnurlResponse?.reason` is set, it should be in the same Alert as the previous one, probably on the second argument (the body of the message is the second argument, first is the title)

Reply:
Ahh... copy pasta side effects